src/components/Header/Header.elements.js

```diff
@@ -1,42 +1,81 @@
 import styled from 'styled-components';
 import { Link } from 'react-router-dom';
+import { GiGolfTee, GiGolfFlag } from 'react-icons/gi'
+import { VscAccount } from 'react-icons/vsc'
 
 export const HeaderContainer = styled.header`
   background: #59A371;
   height: 80px;
   width: 100%;
   display: flex;
-  justify-content: space-between;
+  justify-content: center;
   align-items: center;
   font-size: 2rem;
   position: sticky;
   top: 0;
   z-index: 999;
+`;
 
-  &:hover {
-    color: red;
-  }
+export const HeaderWrapper = styled.nav`
+  display: flex;
+  justify-content: space-between;
+  height: 80px;
+  z-index: 1;
+  width: 100%;
+  max-width: 1600px;
+  margin-right: auto;
+  margin-left: auto;
+  padding-right: 50px;
+  padding-left: 50px;
+
+@media screen and (max-width: 991px) {
```

Comment:
Not sure if we need these media queries, could another solution be switching the values to em or %?